lab_1/main.py

```diff
@@ -36,7 +44,9 @@ def calculate_frequencies(tokens: list) -> dict:
     e.g. tokens = ['weather', 'sunny', 'man', 'happy']
     --> {'weather': 1, 'sunny': 1, 'man': 1, 'happy': 1}
     """
-    pass
+    tokens = ['weather', 'sunny', 'man', 'happy']
```

Comment:
You should not define the `tokens` here - they are coming as an argument.
And you should count the frequencies that are in the passed `tokens` list.